### ISSUE TITLE
(buffer @"") doesn't act as @""

### ISSUE THREAD

**Issue by Saikyun** [janet-lang/janet#530]
On commit ca7c5b8b10d28e1351598e24567e990e3e1564f2
```
(peg/match '(any 1) (buffer @"")) #=> nil
(peg/match '(any 1) @"") #=> @[]
```

**Comment by bakpakin** [comment]
Fixed in eab5f67c5c48393c858f16a1975af773b795d2b9 - sometimes a buffer was created with 0 length that did not have any backing memory (data pointer was NULL). While this was considered valid, it makes lots of manipulations with byte sequence harder so a minimum backing buffer size was added.

**Comment by Saikyun** [comment]
Cool, good job. :)

Skickat från min iPhone
